charts/oes/values.yaml

```diff
@@ -867,12 +867,12 @@ spinnaker:
 
   # Initialize gitOps style Halyard
   gitopsHalyard:
-    enabled: false
+    enabled: true
     mTLS:
       enabled: false   # Enable mTLS for Spinnaker Services and SSL for Deck and Gate
       deckIngressHost: spindeck.{{ .Release.Name }}.domain.com  # Used by OES to create ingress for deck
       gateIngressHost: spingate.{{ .Release.Name }}.domain.com  # Used by OES to create ingress for gate
-    repo-type: git # git, S3, vault, only git support at this point
+    repotype: git # git, S3, vault, only git support at this point
```

Comment:
It should either be camelcase or something that's separated by dash or underscore. e.g repoType or repo-type or repo_type.
Let's go with **repoType**

Reply:
changes done